src/video/ngage/SDL_ngageevents.cpp

```diff
@@ -77,7 +77,7 @@ static SDL_Scancode ConvertScancode(_THIS, int key)
     switch(key)
     {
         case EStdKeyBackspace:    // Clear key
-            keycode = SDLK_BACKSPACE;
+            keycode = SDLK_CLEAR;
```

Comment:
I think this might be an actual backspace key on devices with a full keyboard, so it might be better to leave this one as is.

Reply:
Does that really make a difference? If I am aiming at several devices or/and platforms, I can simply assign several keycodes in my programme.

That's what I did in my N-Gage Wordle clone.